src/main/resources/modernizer.xml

```diff
@@ -595,6 +595,18 @@ violation names use the same format that javap emits.
   </violation>
 
   <violation>
+    <name>com/google/common/base/Preconditions.checkNotNull:(Ljava/lang/Object;)Ljava/lang/Object;</name>
+    <version>1.7</version>
+    <comment>Prefer java.util.Objects.requireNonNull(Object)</comment>
+  </violation>
+
+  <violation>
+    <name>com/google/common/base/Preconditions.checkNotNull:(Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;</name>
```

Comment:
Sorry -- I was looking at the AssertJ source not the Guava source.  Please disregard.